implementations/rust/ockam/ockam_command/src/identity/mod.rs

```diff
@@ -83,14 +83,21 @@ fn create_default_identity(opts: &CommandGlobalOpts) {
     let create_command = CreateCommand::new(default.into(), None);
     create_command.run(opts.clone().set_quiet());
 
+    // Retrieve the identifier if available
+    // Otherwise, use the name of the identity
+    let identity = match opts.state.identities.get(default) {
```

Comment:
Even though we are hiding the term`identifier` to the user, I think it would be better to name this variable as such to avoid confusions.